### DISCUSSION TITLE
Where can one acquire sample CPHD?

### DISCUSSION THREAD

**Discussion by eleehiga** [ngageoint/six-library#717] [Q&A] [answered]

**Comment by eleehiga** [comment]
Some sample CPHD data is here I think: https://www.sdms.afrl.af.mil/index.php?collection=gmti. You guys should update the docs with the link I put here.

EDIT: Sarpy nor Matlab SAR works with this data

**Comment by eleehiga** [comment]
There is CPHD

    https://rdr.ucl.ac.uk/articles/dataset/T-72_SAR_data_dome_Remote_Sensing_0_25_no_PSF_/20681836
    Simulated CPHD files.
 
From https://github.com/ngageoint/sarpy/tree/master/tests. But is there more CPHD data and even nonsimulated ones?

**Comment by eleehiga** [comment]
There is also data here: Gotcha Volumetric SAR Data Set Overview: https://www.sdms.afrl.af.mil/index.php?collection=gotcha. MATLAB SAR supports this but not sarpy

**Reply by HandsomeLee-9966** [reply]
This webpage now can not be linked. Are any other ways to obtain this dataset? Thank you very much!

**Comment by eleehiga** [comment]
But is there even more CPHD data out there?

**Comment by vmartinlac** [comment, marked as answer]
CPHD can be found among Capella Open Data :
https://radiantearth.github.io/stac-browser/#/external/capella-open-data.s3.us-west-2.amazonaws.com/stac/capella-open-data-by-product-type/capella-open-data-cphd/collection.json

**Comment by eleehiga** [comment]
Thank you @vmartinlac
